ansible/roles/ocp-workload-quarkus-cafe-demo-role/readme.adoc

```diff
@@ -0,0 +1,110 @@
+= ocp-workload-quarkus-cafe-demo-role - Quarkus Cafe Workload Role
+= This role is deprecated, please check the new role here link:https://github.com/redhat-cop/agnosticd/tree/development/ansible/roles_ocp_workloads/ocp4_workload_example
+
+== Role overview
+
+* This is a working no-op role that can be used to develop new ocp-workload roles. It consists of the following playbooks:
+** Playbook: link:./tasks/pre_workload.yml[pre_workload.yml] - Sets up an
```

Comment:
This is a tasks file, not a playbook.

Reply:
Did you need me to add project quota to the pre_workload.yml?